etc/di.xml

```diff
@@ -560,6 +560,7 @@
                 <item name="transaction" xsi:type="string">Adyen\Payment\Gateway\Request\CcAuthorizationDataBuilder</item>
                 <item name="vault" xsi:type="string">Adyen\Payment\Gateway\Request\VaultDataBuilder</item>
                 <item name="threeds2" xsi:type="string">Adyen\Payment\Gateway\Request\ThreeDS2DataBuilder</item>
+                <item name="redirect" xsi:type="string">Adyen\Payment\Gateway\Request\RedirectDataBuilder</item>
```

Comment:
Should this be added to `AdyenPaymentOneclickAuthorizeRequest` too? Or oneclick will be handled in another ticket?

Reply:
@Aleffio I think you're right. I also added it to `AdyenPaymentCcVaultAuthorizeRequest` now

Reply:
I don't think it's necessary with Vault, as it uses RECURRING tokens, and they should never trigger 3ds1 or 3ds2 as the shopper may not be present, but maybe double check if it's still the case 👍